src/components/OurValues/OurValues.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import ConsumeIcon from "../../Icons/ConsumeIcon";
+import WasteIcon from "../../Icons/WasteIcon";
+import PlanetIcon from "../../Icons/PlanetIcon";
+import DonateIcon from "../../Icons/DonateIcon";
+import QualityIcon from "../../Icons/QualityIcon";
+import AwarnessIcon from "../../Icons/AwarnessIcon";
+
+function OurValues() {
+  return (
+    <div className="flex flex-row bg-white w-full pt-4 pb-10">
+      <p className="font-semibold text-3xl tracking-tighter text-bric ml-20 mt-10">
+        Our Values
+      </p>
+      <div className="flex flex-col mr-60 mt-20 ">
```

Comment:
you shouldn't add margin-right to each item to make them look like a grid, you can achieve that using flex-box wrap and add space between flex item.

read here
https://css-tricks.com/dont-overthink-flexbox-grids/

and here
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Relationship_of_Grid_Layout